### ISSUE TITLE
msvc errors on every build (platform.h)

### ISSUE THREAD

**Issue by marehr** [seqan/seqan#2205]
Close if there already exists a ticket.

```c++
    46>Done Building Project "C:\jenkins\ws\seqan_win\1da14ac5\build-develop\demos\demo_dox_graph_algorithms_longest_increasing_subsequence.vcxproj" (default targets) -- FAILED.
    45>C:\jenkins\ws\seqan_win\1da14ac5\checkout-develop\include\seqan/platform.h(480): fatal error C1012: unmatched parenthesis: missing ')' 
```

https://github.com/seqan/seqan/blob/028fd79c6e02b7255c95d390b965d922aa874bb6/include/seqan/platform.h#L479-L483

**Comment by marehr** [comment]
Okay is already fixed! I have to rebase my branch.

**Comment by h-2** [comment]
Is your branch up to date? This was fixed in #2196
